assets/css/index.css

```diff
@@ -0,0 +1,163 @@
+*{
+	box-sizing: border-box;
+}
+
+body{
+	font: 12px/18px "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+
+}
+.hero-section a p{
+	color: #fff;
+	font-size: 1.5rem;
+	font-family: 'Nanum Gothic', sans-serif;
+}
+
+.hero-section{
+	position: relative;
+	width: 100%;
+	height: 550px;
+	background-image: url('../images/your_verse_hero.jpg');
+	background-position: 90% 100%;
+	background-repeat: no-repeat;
+	z-index: 1;
+}
+.hero-section ul{
+	z-index: 9998;
+	display: flex;
+	flex-direction: row;
+	position: absolute;
+	top: 25px;
+	width: 1000px;
+	margin: 0 auto;
+	border-radius: 4px;
+	padding: 0 5px;
+	border-radius: 4px;
+	background: -webkit-gradient(linear, 0 0, 0 100%, from(rgba(0, 0, 0, .2)), color-stop(0.05, rgba(0, 0, 0, 0)), color-stop(0.97, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, .45))), -webkit-gradient(linear, 0 0, 100% 0, from(rgba(0, 0, 0, .2)), color-stop(0.002, rgba(0, 0, 0, 0)), color-stop(0.998, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, .2))), -webkit-gradient(linear, 0 100%, 0 0, from(#666), color-stop(0.5, #5e5e5e), color-stop(0.51, #707070), to(#808080));
+}
+.hero-section nav{
+	border-radius: 4px;
+}
+
+.hero-section li{
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	height: 36px;
+	width: 25%;
+	list-style:  none;
+	border-right: solid 1px black;
+}
+
+.hero-section li:last-child{
+	display: flex;
+	flex-direction: row;
+	border-right: 0px;
+	position: relative;
+}
+.hero-section li i.fa-search:last-child{
+	position: relative;
+	left: 15px;
+	top:1.459px;
+	
+}
+.hero-section a{
+	display: block;
+	text-decoration: none;
+	text-align: center;
+	color: white;
+}
+input{
+	border-radius: 14px;
+	border: 1px solid black;
+}
+li:last-child span {
+	position: relative;
+	left: 10;
+}
+
+.img{
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+}
+.img img{
+	margin-bottom: 20px;
+}
+
+.Phone-list{
+	display: flex;
+	flex-direction: row;
+}
+.list1{
+	height: 200px;
+	width: 25%;	
+	background-image: url('../images/promo_iphone5s.jpg');
+	background-position: 50% 50%;
+}
+.list2{
```

Comment:
Repeated style `
height: 200px;
width: 25%;
background-position: 50% 50%;` for `.list1`, `.list2`, `.list3` and `.list4`

I recommend using one class for styling that repeats this many times.